standard/lexical-structure.md

```diff
@@ -64,18 +64,17 @@ The productions for *simple_name* ([§11.7.4](expressions.md#1174-simple-names))
 >
 > *end example*
 
-If a sequence of tokens can be parsed (in context) as a *simple_name* ([§11.7.4](expressions.md#1174-simple-names)), *member_access* ([§11.7.6](expressions.md#1176-member-access)), or *pointer_member_access* ([§22.6.3](unsafe-code.md#2263-pointer-member-access)) ending with a *type_argument_list* ([§8.4.2](types.md#842-type-arguments)), the token immediately following the closing `>` token is examined. If it is one of
+If a sequence of tokens can be parsed (in context) as a *simple_name* ([§11.7.4](expressions.md#1174-simple-names)), *member_access* ([§11.7.6](expressions.md#1176-member-access)), or *pointer_member_access* ([§22.6.3](unsafe-code.md#2263-pointer-member-access)) ending with a *type_argument_list* ([§8.4.2](types.md#842-type-arguments)), the token immediately following the closing `>` token is examined. If it is one of the following:
```

Comment:
This disambiguation is not correct.  The correct disambiguation rule appears in #61.  This rule improperly changes the meaning of `M(a < b > c)` from its current meaning (a syntactically valid chain of comparison operators) into a syntax error.  See [HERE](https://sharplab.io/#v2:CYLg1APgAgTABAQQLACgDeq5blAzDgFjgFkBDASwDsAKASk2wxWxbgA05S4BeOSgUwDu7OgG4GrdnABGPTuOaSpAYzmkFSktS4AeGXAB8cZbQ3YAvhMJaOAD1pw0cSyhepY7VExZ4cARgA2KQB7AAd+ACdSABdgiLgdag4AG34AM2iAGikI8gBzAAtoh24jVIyzLF8oQJDwqNj4gyS4cqyc/KKSsvTohRcgA) for a test that demonstrates that this is not correct.

Reply:
See https://github.com/dotnet/csharpstandard/pull/61/files#r869861306

Reply:
Brought in the changes from #61 (which I believe can be closed without merging).